docs/getting-started/playground-start.md

```diff
@@ -0,0 +1,105 @@
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A Gno.land compatible wallet**
+  - Currently, [Adena](https://www.adena.app/) is the go-to wallet for
+Gno.land, with more wallets being introduced in the future.
+
+## Playground Features
+
+To get started, visit the Playground at [play.gno.land](https://play.gno.land). You will be greeted with a
+simple `package.gno` file:
+
+![default_playground](../assets/getting-started/playground/default_playground.png)
+
+The Playground has the following features:
+- `Share` - Generate a unique, short, and shareable identifier for your Gno code.
+- `Deploy` - Connect your wallet and publish your code on Gno.land.
+- `Format` - Automatically adjust your Gno code's structure and style for optimal readability and consistency.
+- `Run` - Execute a particular expression within your code to validate its functionality and output.
+- `Test` - Execute predefined tests to verify your code's integrity and ensure it meets expected outcomes.
+- `REPL` - Experiment and troubleshoot in real-time using the GnoVM with interactive REPL features.
+
+Let's dive into each of the Playground features.
+
+### Share
+
+The **Share** feature provides users with a permanent shortlink to the current
+Gno code in the playground, making it a simple and easy way to do code-sharing.
+Links created via the **Share** feature initially set to expire after 5 years,
+ensuring the shared code remains accessible over an extended period.
+
+### Deploy
+
+The **Deploy** feature allows users to seamlessly deploy their Gno code to the 
+chain. After connecting a Gno.land wallet, users can select their desired 
+package path and network for deployment.
+
+![default_deploy](../assets/getting-started/playground/default_deploy.png)
+
+After inputting your desired package path, you can select the network you would 
+like to deploy to, such as [Portal Loop](../concepts/portal-loop.md) or local,
+and click deploy.
+
+:::info
+The Playground will automatically provide enough test tokens to cover the gas 
+cost at the time of deployment, removing the need for using a faucet.
+:::
+
+### Format
+
+The **Format** feature utilizes the Monaco editor and
+[`gofmt`](https://pkg.go.dev/cmd/gofmt) to automatically refine and standardize 
+your Gno code's syntax.
+
+### Run
+
+The **Run** feature will allow you to run an expression on your Gno code. Take the following code
+for an example:
+
+[![run_example](../assets/getting-started/playground/run.png)](https://play.gno.land/p/nBq2W8drjMy)
+
+Running `println(Render("Gnopher"))` will display the following output:
+
+```bash
+Hello Gnopher!
+```
+
+View the code [here](https://play.gno.land/p/nBq2W8drjMy).
```

Comment:
Why is there no mention in this section that you can run actual functions?

Reply:
Because running a normal function that does not have a `print` method inside, or just not printing the output of a function, will just give you an empty terminal. This feature is in discussions to be improved with DevX.

Reply:
You can get the return value by running `Run` on the Playground UI, so I'm not sure what the confusion is about

Reply:
Here's what I'm talking about: https://www.loom.com/share/3ee22ee79ee24677935703ba33f42c5d?sid=52c89403-078e-4c96-bed7-2ac779a2f801

Reply:
If I'm not going crazy, calling `Run` would give you the output of the method:
```shell
("Hello" string)
```

Reply:
Linked issue for reference: https://github.com/gnostudio/studio/issues/581